tests/lib/rule-tester/generate-snapshot.js

```diff
@@ -0,0 +1,58 @@
+"use strict";
+
+const path = require("path");
+const { expect, use } = require("chai");
+const chaiJestSnapshot = require("chai-jest-snapshot");
+const { RuleTester } = require("../../../lib/rule-tester");
+
+use(chaiJestSnapshot);
+
+describe("RuleTester:generateSnapshot", () => {
+    let ruleTester;
+
+    // eslint-disable-next-line prefer-arrow-callback
+    before(function() {
```

Comment:
https://github.com/suchipi/chai-jest-snapshot
```
Note: do not use an arrow function for the beforeEach as these will not receive the correct this value provided by mocha.
```

Can't find any better snapshot test for mocha and chai.
It's not hard to have a custom snapshot function anyway.